tests/test_setpoint_constraints.py

```diff
@@ -65,11 +65,6 @@ def test_run_small_ates_timed_setpoints_2_changes(self):
         Checks:
         - That the setpoint does change twice over window length if 2 is specified
 
-        Missing:
-        - This test runs extremely slow in current master, should be fixed!
```

Comment:
was this fixed?

Reply:
Yes as in it now runs fast, but we did not completely debug what was the cause